ext/native-decls/DeleteEntity.md

```diff
@@ -1,14 +1,16 @@
----
-ns: CFX
-apiset: server
----
-## DELETE_ENTITY
-
-```c
-void DELETE_ENTITY(Entity entity);
-```
-
-Deletes the specified entity.
-
-## Parameters
-* **entity**: The entity to delete.
+---

```

Comment:
Are the line changes caused by the encoding change?

Reply:
Yea something weird with my editor I'll recommit with white space changes removed

Reply:
Should be fixed